system/lib/emmalloc.cpp

```diff
@@ -80,6 +80,10 @@ extern "C"
 // in the size field. I.e. for free regions, the size field is odd, and for used regions, the size field reads even.
 #define FREE_REGION_FLAG 0x1u
 
+// Attempts to malloc() more than this many bytes would cause an internal wraparound,
+// therefore allocations larger than this are short-circuited immediately on entry.
```

Comment:
maybe explain why `C7`?

Reply:
Not sure what to add to the existing comment. Reworded it a little.